clippy.toml

```diff
@@ -0,0 +1,10 @@
+cyclomatic-complexity-threshold = 30
+doc-valid-idents = [
+  "MiB", "GiB", "TiB", "PiB", "EiB",
+  "DirectX", "OpenGL", "TrueType",
+  "GPLv2", "GPLv3",
+  "GitHub",
+  "IPv4", "IPv6",
+  "JavaScript", "NaN", "OAuth",
+  "SQLite", "PostgreSQL"
```

Comment:
Ah, I see. We should probably add `MySQL` then. Do we actually need the two lines above this one? `NaN` probably should be in backticks when it appears actually.